src/InputForm.tsx

```diff
@@ -0,0 +1,31 @@
+import { useState } from 'react';
+
+interface InputFormType {
+  setAddTodoIsClicked: React.Dispatch<React.SetStateAction<boolean>>,
+}
+
+export const InputForm: React.FC<InputFormType> = ({ setAddTodoIsClicked }) => {
+  const [inputValue, setInputValue] = useState('');
+
+  const handleInputChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    setInputValue(event.target.value);
+  };
+
+  const handleFormSubmit = (event: React.FormEvent<HTMLFormElement>) => {
+    event.preventDefault();
+    setAddTodoIsClicked(true);
```

Comment:
Instead of just setting a flag on form submit, you should also pass the actual input value to the parent component to handle the task creation. But you can remove it for now, you don't need to implement this feature in the part of the task